backend/dashboard/views/pages/Accounts.vue

```diff
@@ -0,0 +1,170 @@
+<template lang='pug'>
+page-template
+  template(#title='') {{ L('Create a new instance') }}
+
+  section.c-create-instance-section
+    form.c-forms(@submit.prevent='')
+      .field
+        InfoCard.c-info-card(:heading='L("Note")')
+          | In order to create a new Application Instance, you’ll need a domain name. If you don’t already have one,
+          |  you can find an available one and register for a fee with a hosting website and then proceed to create your instance.
+
+      .field
+        i18n.label(tag='label') Software / Application
+
+        dropdown.c-type-dropdown(
+          :class='{ "is-error": $v.form.application.$error }'
+          :options='ephemeral.fakeApplicationOptions'
+          v-error:application=''
+          :disable-click-away='true'
+          @select='onDropdownSelect'
+          @blur='updateField("application", form.application)'
+        )
+
+      .field
+        StyledInput(
+          :class='{ "is-error": $v.form.instanceName.$error }'
+          v-model='form.instanceName'
+          v-error:instanceName=''
+          :label='L("Instance name")'
+          :placeholder='L("Enter instance name")'
+          @blur='updateField("instanceName", form.instanceName)'
+        )
+
+        i18n.helper.c-helper A short alphanumeric name to identify your instance internally. It cannot be changed later and must be unique.
+
+      .field
+        StyledInput(
+          :class='{ "is-error": $v.form.displayName.$error }'
+          v-model='form.displayName'
+          v-error:displayName=''
+          :label='L("Display name")'
+          :placeholder='L("Enter display name")'
+          @blur='updateField("displayName", form.displayName)'
+        )
+
+        i18n.helper.c-helper Used on this dashboard to represent your instance.
+
+      .field.mb-15
+        StyledInput(
+          :class='{ "is-error": $v.form.domain.$error }'
+          v-error:domain=''
+          v-model='form.domain'
+          :label='L("Domain")'
+          :placeholder='L("Enter domain")'
+          @blur='updateField("domain", form.domain)'
+        )
+
+        i18n.helper.c-helper The domain name (or subdomain) where you’ll host your instance. It cannot be changed later.
+
+      fieldset.field
+        legend.label {{ L('Allow unencrypted data on contracts?') }}
+
+        label.radio
+          input.input(
+            type='radio'
+            name='allowUnencrypted'
+            value='yes'
+            v-model='form.allowUnencryptedData'
+          )
+          i18n Allow unencrypted contracts
+
+        label.radio
+          input.input(
+            type='radio'
+            name='allowUnencrypted'
+            value='no'
+            v-model='form.allowUnencryptedData'
+          )
+          i18n Don’t allow unencrypted contracts
```

Comment:
Can there be a question mark with a tooltip or small label or something that explains why this is useful? E.g. something like:

```
If you don't want to moderate public data you should disable unencrypted contract data.
```